docs/conf.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/catalog"))
+sys.path.insert(0, os.path.abspath("../src/dasmon_app/"))
+sys.path.insert(0, os.path.abspath("../src/webmon_app/"))
+sys.path.insert(0, os.path.abspath("../src/workflow_app/"))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Web Monitor"
+copyright = "2022, SCSE@ORNL"
+author = "SCSE@ORNL"
+
+# The short X.Y version
+version = ""
```

Comment:
`version` and `release` can be added to the [bumversion configuration in `setup.cfg`](https://github.com/neutrons/data_workflow/blob/next/setup.cfg#L1-L4)

Reply:
do I manually add them in `conf.py` or is there a better way to do it?